gateway/src/main/java/eu/arrowhead/core/gateway/quartz/RelayConnectionRemovalTaskConfig.java

```diff
@@ -0,0 +1,97 @@
+/********************************************************************************
+ * Copyright (c) 2021 AITIA
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   AITIA - implementation
+ *   Arrowhead Consortia - conceptualization
+ ********************************************************************************/
+
+package eu.arrowhead.core.gateway.quartz;
+
+import java.util.Properties;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.quartz.SimpleTrigger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.quartz.JobDetailFactoryBean;
+import org.springframework.scheduling.quartz.SchedulerFactoryBean;
+import org.springframework.scheduling.quartz.SimpleTriggerFactoryBean;
+
+import eu.arrowhead.common.CoreCommonConstants;
+import eu.arrowhead.common.quartz.AutoWiringSpringBeanQuartzTaskFactory;
+
+@Configuration
+public class RelayConnectionRemovalTaskConfig {
+	
+	//=================================================================================================
+	// members
+	
+	private Logger logger = LogManager.getLogger(RelayConnectionRemovalTaskConfig.class);
+	
+	@Autowired
+	private ApplicationContext applicationContext;
+
+	public static final int SCHEDULER_INTERVAL = 10;
+	
+	private static final int SCHEDULER_DELAY = 2;
+	private static final String NUM_OF_THREADS = "1";
```

Comment:
minor issue, but this should be int. When you use it, you should cast to string if necessary.